cockroach-data-local/temp-dirs-record.txt

```diff
@@ -0,0 +1 @@
+/home/kingmoses/go/src/github.com/decred/dcrdata/cockroach-data-local/cockroach-temp007727042
```

Comment:
Please remove this and cockroach-data-local/cockroach-temp007727042/TEMP_DIR.LOCK